### PR TITLE
Luis swagger fixes

### PR DESCRIPTION
**Pull request by diberry**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json [hunk 1]
```diff
@@ -2605,7 +2648,7 @@
     },
     "/apps/{appId}/querylogs": {
       "get": {
-        "description": "Gets the query logs of the past month for the application.",
+        "description": "Gets thelogs of the past month's endpoint queries for the application.",
```
Comment (nschonni) [inline specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json]: thelogs -> the logs

File: specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json [hunk 2]
```diff
@@ -8027,14 +8076,14 @@
...
+          "description": "An interchangeable phrase list feature serves as a list of synonyms for training. A non-exchangeable phrase list serves as separate features for training. So, if your non-interchangable phrase list contains 5 phrases, they will be mapped to 5 separate features. You can think of the non-interchangeable phrase list as an additional bag of words to add to LUIS existing vocabulary features. It is used as a lexicon lookup feature where its value is 1 if the lexicon contains a given word or 0 if it doesn’t.  Default value is true.",
```
Comment (nschonni) [inline specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json]: interchangable -> interchangeable

File: specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json [hunk 3]
```diff
@@ -8179,7 +8228,7 @@
...
```
Comment (nschonni) [inline specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json]: interchangable -> interchangeable

File: specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json [hunk 4]
```diff
@@ -9214,7 +9263,7 @@
...
+          "description": "An exchangeable phrase list feature are serves as single feature to the LUIS underlying training algorithm. It is used as a lexicon lookup feature where its value is 1 if the lexicon contains a given word or 0 if it doesn’t. Think of an exchangeable as a synonyms list. A non-exchangeable phrase list feature has all the phrases in the list serve as separate features to the underlying training algorithm. So, if you your phrase list feature contains 5 phrases, they will be mapped to 5 separate features. You can think of the non-exchangeable phrase list feature as an additional bag of words that you are willing to add to LUIS existing vocabulary features. Think of a non-exchangeable as set of different words. Default value is true.",,
```
Comment (nschonni) [inline specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json]: Double comma at the end

Comment (diberry) [reply specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json]: @nschonni what tool did you use to find these issues?

Comment (nschonni) [reply specification/cognitiveservices/data-plane/LUIS/Authoring/stable/v2.0/LUIS-Authoring.json]: Opening the content in VS Code caught the double comma at the end as an error. I've got an running PR to add a cSpell dictionary with #4836. Right now, I just open that branch and past the code into an existing file to see the issues though.

**Comment by jhendrixMSFT** [comment]
Please note this contains breaking changes in a swagger marked as stable which is discouraged.

**Comment by shahabhijeet** [comment]
@jhendrixMSFT .NET SDK does not support multiple RPs as of now.
I think we should have two separate PRs for each RP.
If one SDK generation fails and other succeeds, it's challenging to let the user know what exactly failed as there are only 1 job created for 1 RP.
But if you still would like to merge, go ahead, .NET SDK has still not enabled multiple RPs.
Plus this is data plane SDK

**Comment by diberry** [comment]
@jhendrixMSFT @shahabhijeet @cahann @nebadr 

This PR has changes from the service team, provided by @nebadr and changes to the descriptions I provided as docs member.  

I'll remove the runtime changes so it is only 1 file being changed. 

Are there specific issues about the service team's changes that you need addressed?

**Comment by diberry** [comment]
Runtime broken out into different PR https://github.com/Azure/azure-rest-api-specs/pull/5424

**Comment by adxsdknet** [comment]
# Automation for azure-sdk-for-net 
A PR has been created for you: 
https://github.com/azure/azure-sdk-for-net/pull/5511 
.NET SDK Commits: 
https://github.com/adxsdknet/azure-sdk-for-net/pull/5511/commits/39f8c637bc154df244074c809cd8c1594c67f7b0

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**